client/src/common/notebook.ts

```diff
@@ -413,12 +413,26 @@ export type NotebookDocumentMiddleware = {
 };
 
 export interface NotebookDocumentSyncFeatureShape {
+	onOpenNotificationSent: vscode.Event<vscode.NotebookDocument>;
+	onChangeNotificationSent: vscode.Event<VNotebookDocumentChangeEvent>;
+	onCloseNotificationSent: vscode.Event<vscode.NotebookDocument>;
+	onSaveNotificationSent: vscode.Event<vscode.NotebookDocument>;
 	sendDidOpenNotebookDocument(notebookDocument: vscode.NotebookDocument): Promise<void>;
 	sendDidSaveNotebookDocument(notebookDocument: vscode.NotebookDocument): Promise<void>;
 	sendDidChangeNotebookDocument(event: VNotebookDocumentChangeEvent): Promise<void>;
 	sendDidCloseNotebookDocument(notebookDocument: vscode.NotebookDocument): Promise<void>;
 }
 
+export interface NotificationNotebookSendEvent<P> {
```

Comment:
Why do we need the two interface? They seem to be unused to me.

Reply:
Not needed. Leftover code from a different refactoring that I forgot to remove.